graphql-schemas/schema.yml

```diff
@@ -3218,6 +3219,41 @@ confs:
   - { name: origin, type: string, isRequired: true }
   - { name: records, type: DnsRecord_v1, isList: true }
 
+- name: CloudflareDnsRecordDataSettings_v1
+  fields:
+  - { name: protocol, type: int }
+  - { name: public_key, type: string }
+  - { name: digest_type, type: int }
+  - { name: digest, type: string }
+
+- name: CloudflareDnsRecord_v1
+  datafile: /cloudflare/dns-record-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: name, type: string, isRequired: true }
+  - { name: type, type: string, isRequired: true}
+  - { name: ttl, type: int, isRequired: true}
+  - { name: value, type: string , isRequired: true}
+  - { name: data, type: CloudflareDnsRecordDataSettings_v1 }
+  - { name: proxied, type: boolean }
+  - { name: priority, type: int }
+
+
+- name: CloudflareDnsZone_v1
+  datafile: /cloudflare/dns-zone-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: identifier, type: string, isRequired: true }
+  - { name: account, type: CloudflareAccount_v1, isRequired: true }
+  - { name: zone, type: string, isRequired: true }
+  - { name: type, type: string }
+  - { name: data, type: string }
```

Comment:
What is this field for?

Reply:
This is a field for DS and DNSKEY records specifically, see https://github.com/app-sre/qontract-schemas/pull/367#discussion_r1068495774

Reply:
I see a `data` field in the DNS record schema for DS and DNSKEY records but this one here is in the zone schema which don't seem to exist in the provider?

Reply:
Ahh you are right, my mistake. Removing.